metrics/web/test/features/dashboard/presentation/page/dashboard_page_test.dart

```diff
@@ -97,29 +113,111 @@ void main() {
         );
       },
     );
+
+    testWidgets(
+      "displays performance title above the sparkline graph widget",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const _DashboardTestbed());
+        await tester.pumpAndSettle();
+
+        final performanceMetricWidgetCenter = tester.getCenter(
+          find.byType(SparklineGraph),
+        );
+
+        final performanceTitleCenter = tester.getCenter(
+          find.descendant(
+              of: find.byType(Expanded),
+              matching: find.text(DashboardStrings.performance)),
+        );
+
+        expect(performanceMetricWidgetCenter.dx, performanceTitleCenter.dx);
+      },
+    );
+
+    testWidgets(
+      "displays builds title above the build number text metric widget",
```

Comment:
Maybe we should create MetricsTable widget to make sure that all of the table functionality is localized there?